ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/logger/TestSlf4jAuditLogger.java

```diff
@@ -54,7 +54,7 @@
 @RunWith(MockitoJUnitRunner.StrictStubs.class)
 public class TestSlf4jAuditLogger
 {
-    private static final String DEFAULT_LOG_FORMAT = "client:'${CLIENT_IP}'|user:'${USER}'{?|batchId:'${BATCH_ID}'?}|status:'${STATUS}'|operation:'${OPERATION}'";
+    private static final String DEFAULT_LOG_FORMAT = "client:'${CLIENT_IP}'|user:'${USER}'{?|batchId:'${BATCH_ID}'?}|status:'${STATUS}'|operation:'${OPERATION}'{?|subject:'${SUBJECT}'?}";
```

Comment:
This constant is used to verify what is actually the defautl log format in ecAudit, so we shouldn't change it. There is a test case below for testing custom formats where this could be added instead.